Documentation/ApiOverview/Workspaces/Index.rst

```diff
@@ -136,11 +136,11 @@ Frontend implementation guidelines
          This is how simple it is to use this record in your frontend plugins
          when you do queries directly (not using API functions already using
          them)::
-
-            $res = $GLOBALS['TYPO3_DB']->exec_SELECTquery(...);
-            while (($row = $GLOBALS['TYPO3_DB']->sql_fetch_assoc($res))) {
+         
+            $statement = $queryBuilder->execute();
+            while ($row = $statement->fetch()) {
```

Comment:
The `$result` is iterable so we can actually use `foreach ($result as $row)`

Reply:
nice to know, because this appears to be missing in the Query Builder documentation. Maybe I will add this next.